frontend/src/scenes/trends/viz/ActionsLineGraph.tsx

```diff
@@ -46,11 +46,12 @@ export function ActionsLineGraph({
                         : (point) => {
                               const { dataset, day } = point
                               loadPeople(
-                                  dataset.action || 'session',
+                                  dataset.action,
```

Comment:
Now I'm confused. You went through all this trouble to add the `| 'session'` type, yet now you're just removing the only place that calls this with `'session'`. So then the type is not needed? Or is there a bug here?